.github/workflows/compile.yml

```diff
@@ -0,0 +1,38 @@
+name: compile
+on: [push, workflow_dispatch]
+
+jobs:
+  job:
+    name: ${{ matrix.os }}-${{ github.workflow }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        include:
+          - os: windows-latest
+            triplet: x64-windows
+          - os: ubuntu-latest
+            triplet: x64-linux
+          - os: macos-latest
+            triplet: x64-osx
+    steps:
+      - uses: actions/checkout@v3
+      # Install CMake
+      - uses: lukka/get-cmake@latest
```

Comment:
I wouldn't use `latest` because it's generally dangerous. I'd go with a specific tag and add a dependabot config later to handle updating GH Actions on a monthly basis.

Ideally, it should be requested that they also tag `v3` etc upstream.